### ISSUE TITLE
Problem with the use of element (Android) in Russia

### ISSUE THREAD

**Issue by Parodontitis** [element-hq/element-android#8924]
### Steps to reproduce

1. Start element app.
2. Click on "sing in".

#### What did you expect?
New view to select server and insert credentials.

#### What happened instead?
The circle-arrow turns endlessly.

This only happens when using it in Russia. My assumption is that element is trying to establish a connection to matrix.org/federation at this point, which is not possible in Russia. Hence the endless circle.

But there is no possibility to skip this step and enter own server and user/passwd.

X-check with fluffychat was successful. 

### Your phone model

oneplus 10 pro / Motorola

### Operating system version

Android 13 / 14

### Application version and app store

Element version 1.6.20 from F-Droid

### Homeserver

Synapse

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by bro2020** [comment]
Ask your country's government to stop attacking neighboring countries.  This is the solution to your problem.

**Comment by RomanApps** [comment]
I really don’t get why Russians need Android — go back to button Nokias! Calls? It calls. Texts? It texts. And it’s been working for 15 years without a single update!

**Comment by kononiuk** [comment]
If your country kills and robs, you have problems. Elementary logic

**Comment by Manifoldme** [comment]
> Ask your country's government to stop attacking neighboring countries. This is the solution to your problem.



> I really don’t get why Russians need Android — go back to button Nokias! Calls? It calls. Texts? It texts. And it’s been working for 15 years without a single update!



> If your country kills and robs, you have problems. Elementary logic

There's a thing about tribal insanity, you don't know you're afflicted with it, and so you brag about it in the most inappropriate place at the most inappropriate time, just like the Germans did with the Jews. Divide and rule the f plankton, nothing has changed.

Back to the main topic.
I don't think the issue you raise can be solved by the Synapse developers.
Unfortunately, the problem seems to lie at the regulatory level in Russia, the Western rulers are able to force the companies to build backdoors, so they don't have to ban directly something like basic encryption HTTPS or IPs on DNS level like Russia or China do.
I suggest to get used to own a VPS in a neutral jurisdiction like Hezner servers in Singapore, self-host a VPN, for multi-users I would suggest Netbird and Headscale, for less amount of users just Wireguard, and deploy you Synapse or Dendrite on the same server as well.

**Comment by RomanApps** [comment]
> > Ask your country's government to stop attacking neighboring countries. This is the solution to your problem.
> 
> > I really don’t get why Russians need Android — go back to button Nokias! Calls? It calls. Texts? It texts. And it’s been working for 15 years without a single update!
> 
> > If your country kills and robs, you have problems. Elementary logic
> 
> There's a thing about tribal insanity, you don't know you're afflicted with it, and so you brag about it in the most inappropriate place at the most inappropriate time, just like the Germans did with the Jews. Divide and rule the f plankton, nothing has changed.
> 
> Back to the main topic. I don't think the issue you raise can be solved by the Synapse developers. Unfortunately, the problem seems to lie at the regulatory level in Russia, the Western rulers are able to force the companies to build backdoors, so they don't have to ban directly something like basic encryption HTTPS or IPs on DNS level like Russia or China do. I suggest to get used to own a VPS in a neutral jurisdiction like Hezner servers in Singapore, self-host a VPN, for multi-users I would suggest Netbird and Headscale, for less amount of users just Wireguard, and deploy you Synapse or Dendrite on the same server as well.

The whole civilized world knows how to respond to these kinds of provocations, and spoiler alert: history won’t exactly be on your side. But if you're set on those "tribal" takes, maybe it’s time to join some discussions in the local chats for your tribe, like on ok.ru.

**Comment by kononiuk** [comment]
> There's a thing about tribal insanity, you don't know you're afflicted with it, and so you brag about it in the most inappropriate place at the most inappropriate time, just like the Germans did with the Jews. Divide and rule the f plankton, nothing has changed.
Is it necessary to mention tribes for individuals from the Stone Age? Go look for answers at the Cheburnet, not in the civilised world.

**Comment by kononiuk** [comment]
> There's a thing about tribal insanity, you don't know you're afflicted with it, and so you brag about it in the most inappropriate place at the most inappropriate time, just like the Germans did with the Jews. Divide and rule the f plankton, nothing has changed.

Is it necessary to mention tribes for individuals from the Stone Age? Go look for answers at the Cheburnet, not in the civilised world.

**Comment by ShadowRZ** [comment]
To me it looks like a duplicate of https://github.com/element-hq/element-android/issues/7078

However the conversation in this issue seems pretty unreasonable to me.

**Comment by spaetz** [comment]
Yes, EA first tries matrix.org and only then offers connections to other servers, IIRC. And if matrix.org is blocked by a country, login will not work. Not sure if this will be fixed, given that most attention goes into the Element-X clients nowadays.

**Comment by ara4n** [comment]
dup https://github.com/element-hq/element-android/issues/7078
